titan/features/haart.py

```diff
@@ -121,20 +122,21 @@ def update_agent(self, model: "model.TITAN"):
                         ):
                             self.initiate(model)
 
-            # Go off HAART
-            elif self.active:
+            # Update agents on HAART
+            else:
+                # Go off HAART
                 if model.run_random.random() < haart_params.discontinue:
                     self.active = False
                     self.adherent = False
                     self.remove_agent(self.agent)
-                elif (
+                elif (  # Become non-adherent
```

Comment:
the comments here and on the next one look funny to me inside the parens, could you move them to the lines above in both cases?